src/components/RightSidebar/Description/Description.vue

```diff
@@ -0,0 +1,204 @@
+<!--
+  - @copyright Copyright (c) 2020 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<!-- Description -->
+	<div class="description">
+		<!-- Title of the description section -->
+		<h3 class="description__title">
+			{{ descriptionTitle }}
+		</h3>
+		<!-- Text of the description -->
+		<div class="description__body">
+			<RichContentEditable v-model="descriptionText"
+				class="description__contenteditable"
+				:maxlength="100"
+				:multiline="true"
+				:contenteditable="isEditingDescription"
+				:placeholder="descriptionPlaceholder"
+				@submit="handleSubmitDescription" />
+			<div class="description__buttons">
+				<button
+					v-if="!isEditingDescription"
+					class="description__button"
+					:aria-label="t('spreed','Edit description')"
```

Comment:
"Edit conversation description" for more context ? (if a screen reader was tabbed to this button I'm not sure if the user would know that they are now inside the sidebar)